src/coordinator/storage/local/config.go

```diff
@@ -186,5 +190,5 @@ func (c ClustersStaticConfiguration) NewClusters() (Clusters, error) {
 	}
 
 	return NewClusters(unaggregatedClusterNamespace,
-		aggregatedClusterNamespaces)
+		aggregatedClusterNamespaces...)
```

Comment:
nit: Since we're splitting newlines here anyway, might look better as 
```
NewClusters(
 unaggregatedClusterNamespace,
 aggregatedClusterNamespaces...,
)
```
?